nautobot/dcim/tables/devices.py

```diff
@@ -1122,3 +1129,87 @@ class Meta(BaseTable.Meta):
             "tags",
             "actions",
         )
+
+
+class ControllerTable(BaseTable):
+    """Table for list view."""
+
+    pk = ToggleColumn()
+    name = tables.Column(linkify=True)
+    status = ColoredLabelColumn()
+    location = tables.Column(linkify=True)
+    platform = tables.Column(linkify=True)
+    role = tables.Column(linkify=True)
+    tenant = TenantColumn()
+    external_integration = tables.Column(linkify=True)
+    deployed_controller_device = tables.Column(linkify=True)
+    deployed_controller_group = tables.Column(linkify=True)
+    tags = TagColumn(url_name="dcim:controller_list")
+    actions = ButtonsColumn(Controller)
+
+    class Meta(BaseTable.Meta):
+        """Meta attributes."""
+
+        model = Controller
+        fields = (
+            "pk",
+            "name",
+            "status",
+            "location",
+            "platform",
+            "role",
+            "tenant",
+            "external_integration",
+            "deployed_controller_device",
+            "deployed_controller_group",
+            "tags",
+            "actions",
+        )
+        default_columns = (
+            "pk",
+            "name",
+            "status",
+            "location",
+            "platform",
+            "role",
+            "tenant",
+            "external_integration",
+            "deployed_controller_device",
+            "deployed_controller_group",
+            "actions",
+        )
```

Comment:
That's a lot of default columns - how's the usability here? Should we hide any of these columns by default for a better UX?

Reply:
Any proposal pls?

Reply:
Maybe don't show `external_integration`, `deployed_controller_device`, `deployed_controller_group` by default? Not sure.